README.adoc

```diff
@@ -35,5 +35,5 @@ Know when your SQS messages are ending up in a DLQ.
 link:./modules/rds_overspent_instance_credits/[RDS Overspent Instance Credits]::
 Monitor for RDS instances that have overspent their instance credits and incurred costs.
 
-link:./modules/cloudwatch_too_many_errors/[Cloudwatch Too Many Errors]::
-Monitor for too many errors in the ECS service's cloudwatch logs.
+link:./modules/cloudwatch_log_level_metric_alarm/[Cloudwatch Log Level Metric Alarm]::
```

Comment:
Siste nit her: At det er en _metric_ alarm er irrelevant. Det man ser etter er jo alarm for log level, metricen er en implementasjonsdetalj. I tillegg er det implisit en alarm, siden det er i alarms repoet :D

Så kall det heller `cloudwatch_log_level`!